docker-compose.yaml

```diff
@@ -0,0 +1,130 @@
+version: "3.8"
+services:
+    ollama-server:
+        image: ollama/ollama:0.1.39
+        pull_policy: always
+        restart: always
+        container_name: ollama
+        ports:
+            - "${OLLAMA_PORT}:11434"
```

Comment:
exposing ports to host is not strictly necessary if we are using a bridge network
Should only need to expose for backend and frontend